### PR TITLE
Sdata

### PR DESCRIPTION
**Pull request by FazanaJ**

### CODE REVIEW

**Comment by arthurtilly** [comment]
this pr FUCKING SUCKS!

**Comment by Reonu** [comment]
bruh

**Comment by Reonu** [comment]
can we merge this @thecozies @arthurtilly

**Comment by arthurtilly** [comment]
HOW SHOULD I KNOW

File: Makefile [hunk 1]
```diff
@@ -443,7 +443,9 @@ AS        := $(CROSS)as
 ifeq ($(COMPILER),gcc)
   CC      := $(CROSS)gcc
   $(BUILD_DIR)/actors/%.o:           OPT_FLAGS := -Ofast -mlong-calls
+  $(BUILD_DIR)/levels/%.o:           CFLAGS += -G 0
```
Comment (gheskett) [inline Makefile:446]: I think this is supposed to be actors, not levels

**Review by gheskett — CHANGES_REQUESTED** [review]

**Comment by Reonu** [comment]
closing this PR since apparently the concensus is that it won't get merged, but feel free to re-open it if that changes

**Comment by Reonu** [comment]
My bad, apparently there's a plan to fix this

**Comment by gheskett** [comment]
Can we close this PR a third time? The answer might surprise you!

**Comment by gheskett** [comment]
As of HackerSM64 2.4, this PR has been outdated in favor of this branch: https://github.com/HackerN64/HackerSM64/tree/develop/2.1.0-sdata

No new PR will be opened for this until it is decided to be worth merging into HackerSM64 (if that ever happens).

**Pull request closed without merging**